src/Manifest.js

```diff
@@ -244,6 +244,15 @@ function Manifest(output, path) {
         }
     }
 
+    // Windows codepage
+    // Optional field, only check if present.
+    this._windowsCodepage = "";
+    if (json.xwalk_windows_codepage) {
+        this._windowsCodepage = json.xwalk_windows_codepage;
+    } else {
+        this._windowsCodepage = "1252";
```

Comment:
@kenchris because Windows (WiX) does not support it for strings that are displayed in the installer UI.